docs/extensions.md

```diff
@@ -40,6 +40,7 @@ Following are some prewritten extensions that are supported with Siddhi;
 1. <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-sqs">io sqs</a>
 1. <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-twitter">io twitter</a>
 1. <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-cdc">io cdc</a>
+1. <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-prometheus">io prometheus</a>
```

Comment:
This is an auto-generated doc. Add the extension as a dependency in doc-gen pom, else these will get replaced in a release build

Reply:
Added dependency in b80a3e8eadedf165bcf79f6f03306a0ad9a9321e